websites/V/VRV/dist/metadata.json

```
@@ -13,11 +13,12 @@
 	"service": "VRV",
 	"description": {
 		"en": "VRV is the home of your favorite channels, events, and communities celebrating anime, animation, video games, comics, science fiction, fantasy, and tech.",
+		"fr": "VRV est l'endroit pour vos chaînes, événements and communautés favoris autour des animes, de l'animation, des jeux videos, des comics, de la science fiction, de la fantasie et de la technologie.",
```

```suggestion
		"fr": "VRV est la maison de vos chaînes, événements et communautés préférés célébrant l'anime, l'animation, les jeux vidéo, les bandes dessinées, la science-fiction, la fantaisie et la technologie.",
```

---

A la limite ici on peut mettre `les BD` pour la traduction des comics

```suggestion
		"fr": "VRV est la maison de vos chaînes, événements et communautés préférés célébrant l'anime, l'animation, les jeux vidéo, les BD, la science-fiction, la fantaisie et la technologie.",
```